package.json

```diff
@@ -122,6 +122,8 @@
     "scripts": {
         "clean": "rm -rf ./out",
         "vscode:prepublish": "webpack --mode production",
+        "publish": "npx vsce package && npx vsce publish",
+        "unpublish": "npx vsce unpublish UCLResearchSoftwareDevelopmentGroup.nisaba",
```

Comment:
Honestly, I'm a bit nervous about having this command here, given the fact it can be quite disruptive.  We can accidentally click on this command in VS Code!

Reply:
Ok, let's remove it :)